### PR TITLE
feat: UserTopReader entity and GraphQL queries

### PR DESCRIPTION
**Pull request by omBratteng**
TODO:

- [x] Add tests

AS-657

### CODE REVIEW

File: src/schema/users.ts [hunk 1]
```diff
@@ -1659,6 +1696,29 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         },
       );
     },
+    topReaderBadge: async (
+      _,
+      { id }: { id: string },
+      ctx: AuthContext,
+      info: GraphQLResolveInfo,
+    ) => {
+      const res = await graphorm.query<GQLUserTopReader>(
```
Comment (capJavert) [inline src/schema/users.ts:1705]: you can use `queryOneOrFail`

File: src/graphorm/index.ts [hunk 2]
```diff
@@ -165,6 +165,16 @@ const obj = new GraphORM({
         },
         transform: nullIfNotLoggedIn,
       },
+      topReader: {
+        relation: {
+          isMany: true,
+          customRelation: (_, parentAlias, childAlias, qb): QueryBuilder =>
+            qb
+              .where(`${childAlias}."userId" = ${parentAlias}.id`)
+              .orderBy(`${childAlias}."issuedAt"`, 'DESC')
```
Comment (capJavert) [inline src/graphorm/index.ts:174]: might need an index over `userId` + `issuedAt`

Comment (omBratteng) [reply src/graphorm/index.ts:174]: Will add that

Comment (omBratteng) [reply src/graphorm/index.ts:174]: Added

File: src/schema/users.ts (hunk 1 above)
Comment (omBratteng) [reply src/schema/users.ts]: Changed to use that

**Review by capJavert — APPROVED** [review]

**Pull request merged**